components/browser/session-storage/src/main/java/mozilla/components/browser/session/storage/serialize/BrowserStateReader.kt

```diff
@@ -193,6 +195,7 @@ private fun JsonReader.tabSession(): RecoverableTab? {
             Keys.SESSION_HISTORY_METADATA_SEARCH_TERM -> historyMetadataSearchTerm = nextStringOrNull()
             Keys.SESSION_HISTORY_METADATA_REFERRER_URL -> historyMetadataReferrerUrl = nextStringOrNull()
             Keys.SESSION_LAST_ACCESS -> lastAccess = nextLong()
+            Keys.SESSION_CREATED_AT -> createdAt = nextLongOrNull()
```

Comment:
Is `nextLongOrNull` really needed here? We don't use it for `lastAccess` and we would never save the key with a null value?

Reply:
Not 100% sure it's needed, but I was thinking of the edge case where we have a json of an "old" tab that did not have a `createdAt` field, then app gets updated to look for `createdAt` and doesn't find it -> `IllegalStateException`.

Reply:
If there's no `createdAt` value in the file, then we shouldn't end up here **after** we read the created_at key.